lms/djangoapps/shoppingcart/processors/CyberSource2.py

```diff
@@ -520,6 +525,16 @@ def _get_processor_exception_html(exception):
                 email=payment_support_email
             )
         )
+    elif isinstance(exception, CCProcessorUserDeclined):
+        return _format_error_html(
+            _(
+                u"Sorry! Our payment processor sent us back a message saying that transaction was declined. "
+                u"The items in your shopping cart will exist for future purchase. "
+                u"If you feel that this is in error, please contact us with payment-specific questions at {email}."
```

Comment:
Suggested text:

We're sorry, but this payment has been declined. The items in your shopping cart have been saved.

If you have any questions about this transaction, please contact us at {email].